src/ops/ops.go

```diff
@@ -49,6 +49,7 @@ const (
 type Ops interface {
 	Mkdir(dirName string) error
 	WriteImageToDisk(liveLogger io.Writer, ignitionPath string, device string, extraArgs []string) error
+	WriteImageToLocalDevice(liveLogger io.Writer, ignitionPath string, rhelCoreOSImage string) error
```

Comment:
Any ideas for a better name for this?

Technically all the devices are "local". Maybe "BootDevice"? Is that better?

Reply:
BootDevice seems reasonable.

We are describing the disk we will boot into once the node reboots out of discovery?

Reply:
Yes, but that's true in either case.

The distinction is if we're installing to a device that is currently being used for the rootfs or one that is not.

Reply:
So in both cases they are boot disks, so how about 

`WriteImageToBootDiskWithRootFs` ?

Reply:
Maybe `WriteImageToExistingRoot` to mirror the similar `bootc` command (which might be something I can use) called [`bootc install to-existing-root`](https://containers.github.io/bootc/man/bootc-install-to-existing-root.html)